frontend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "badge-frontend",
```

Comment:
This isn't a blocker for merging this PR, but..

This is a workaround we will probably be stuck with for a while, but presumably at some point down the line we will be able to migrate the frontend from CommonJS to ESM too and then we can remove this.

I think it would be really useful to have an issue with a quick summary of

- How far did you get
- What problems did you encounter
- What are the blockers (is gatsby core the only package we know of with a problem, or do we know of other packages in the frontend tree that would also need an update to unblock us - was tsnode one?)
- If/when we can migrate the frontend to ESM:
  - What hacks can we remove?
  - Roughly what does that process look like?

Even if it is not 100% clear, I guess there is an extent to which that information currently exists in your head (or at least you have a clearer picture of this than anyone else) and it will be useful info at some point in future.

More broadly, I am so relieved you managed to find a way to avoid splitting the dependency tree in two :relieved:

Reply:
Good point will prepare a little issue with some of the stuff in my head. :)

Reply:
Here we go: #6717